doc/RELEASE_NOTES.md

```diff
@@ -86,6 +86,15 @@ The PCP support also allows you to register PCP as a transient tool, where it is
 _**NOTE AS WELL**_: like all the other "tools" the `pbench-agent` supports, the `pcp` tools themselves need to be installed separately on the registered hosts.
 
 
+## CLI Interfaces to the Tool Meister Sub-system
+
+Two new CLI command are provided to start and stop the Tool Meister sub-system, `pbench-tool-meister-start` and `pbench-tool-meister-stop`.  These two interfaces are mostly used by the benchmark convenience scripts provided by the Pbench Agent.
+
+If you were using the `pbench-start-tools`, `pbench-stop-tools`, and `pbench-postprocess-tools` before, you must now invoke `pbench-tool-meister-start` before using the start/stop/postprocess tool interfaces.
+
+Further the `pbench-send-tools` CLI interface has been added to instruct the Tool Meister sub-system when it is time to send the collected data from transient tools to the Tool Data Sink.  The typical sequence is to "start", "stop", "send", "post-process".  Now you can "send" and "post-process" the transient tool data any time before `pbench-tool-meister-stop` is called.
+
```

Comment:
It is deliberate that I am not mentioning `pbench-tool-meister-client`, because the `pbench-start-tools`, `pbench-stop-tools`, and `pbench-postprocess-tools` were used by some in the past (I have never been in favor of exposing those interfaces from the beginning), so I just wanted to be sure that if anybody is using those that would be told to add `pbench-send-tools`.

Since `-start-`, `-stop-`, and `-send-` are really just wrappers around `pbench-tool-meister-client`, I didn't think it was worth mentioning in the release notes.